package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "dictionary-component",
+  "version": "1.0.0",
+  "description": "Dictionary Component for CSV lookups on the eio platform",
+  "author": "elastic.io GmbH",
+  "scripts": {
+    "test": "mocha spec --recursive"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/elasticio/dictionary-component.git"
+  },
+  "devDependencies": {
+    "chai": "4.2.0",
+    "eslint": "6.6.0",
+    "eslint-config-airbnb-base": "14.0.0",
+    "eslint-plugin-import": "2.18.2",
+    "mocha": "6.2.2",
+    "sinon": "7.5.0"
+  },
+  "dependencies": {
+    "elasticio-node": "0.0.9",
+    "elasticio-sailor-nodejs": "2.5.1",
+    "papaparse": "5.1.0"
```

Comment:
Better to use [csv](https://www.npmjs.com/package/csv) library kike in csv-component, if it possible, instead of `papaparse` - to keep one library for working wo csv files in our projects

Reply:
I just looked into the csv library and implementing it would mean changing 100% of the code that is written. The library I have used has some very nice things about it, like automatically determining the delimiter, neat error and header handling, which I have taken advantage of.

If this absolutely must be changed, I would rather open an issue for it? I also just looked at our csv component, and it has a lot of config fields for things that are handled by this library automatically. I think there are a lot of cases that could be simplified by switching that library to the one I have used here, improving the usability of the component. What do you think?